src/components/Questions.tsx

```diff
@@ -53,7 +41,10 @@ export default function Questions() {
         </ListSubheader>
       }
     >
-      <ListItemButton onClick={handleClick}>
+      <ListItemButton
+        selected={selectedIndex === 0}
```

Comment:
does this code do something? just wondered!

Reply:
Its a work in progress;)

Reply:
took away the code that is under work- added the old original thats working for now